src/test/java/seedu/address/model/contact/ContactTest.java

```diff
@@ -72,7 +72,7 @@ public void hasSameFields() {
         // null -> returns false
         assertFalse(ALICE.hasSameFields(null));
 
-        // different fields for (name + nickname), tele handle, email -> false
+        // different fields for (name + nickname), tele handle, and email and  no same president role -> false
```

Comment:
More test cases needed to show isBothPresident -> true
if planning to do in the future can just ignore for now